packages/core/src/components/Table/TableCell/TableCell.tsx

```diff
@@ -93,6 +93,12 @@ const StyledTableCell = (c: any) =>
     }),
   }));
 
+const getSortedColor = (color: string, alpha: string) => {
```

Comment:
The same thing is happening here:

"This way of getting the background color is preferable because we get less re-renders. However, with these changes, the background color is not correct on the first render since we get `undefined`. I think it's because the CSS vars are not yet defined when we try to get the background value."